torch/csrc/jit/script/module.h

```diff
@@ -222,8 +226,9 @@ struct TORCH_API Module {
     }
     return c10::nullopt;
   }
-  c10::optional<Method> find_method(const std::string& name) const {
-    if (const auto fn = class_compilation_unit()->find_function(name)) {
+  c10::optional<Method> find_method(const std::string& basename) const {
```

Comment:
This runs in the user's eval loop, so I am not really happy about having to allocate a new string in order to look up the function. I feel like the ClassType should just be holding onto this mapping directly.